tms_blog/src/components/Button/Button.css

```diff
@@ -0,0 +1,85 @@
+.App {
+  background: #11ccda;
+}
+.ourButtonStyle {
```

Comment:
Основной класс для компонента должен называться точно также как и сам компонент
```suggestion
.button {
```